editor/editor_node.cpp

```diff
@@ -2877,6 +2877,12 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 
 			if (!scene->get_scene_file_path().is_empty()) {
 				String path = scene->get_scene_file_path();
+				String root_name = scene->get_name();
+				root_name = EditorNode::adjust_scene_name_casing(root_name);
```

Comment:
Can be simplified to
```suggestion
				const String root_name = EditorNode::adjust_scene_name_casing(scene->get_name());
```